MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/IsochroneActivity.java

```diff
@@ -80,32 +86,46 @@ protected void onCreate(Bundle savedInstanceState) {
 
     mapView = findViewById(R.id.mapView);
     mapView.onCreate(savedInstanceState);
-    mapView.getMapAsync(mapboxMap -> {
-      this.mapboxMap = mapboxMap;
-      mapboxMap.addOnMapClickListener(IsochroneActivity.this);
-      initIsochroneCenterSymbolLayer();
-
-      // Set the click listener for the floating action button
-      findViewById(R.id.switch_isochrone_style_fab).setOnClickListener(view -> {
-        usePolygon = !usePolygon;
-        redrawIsochroneData(lastSelectedLatLng);
-      });
-
-      Toast.makeText(this, getString(R.string.click_on_map_instruction), Toast.LENGTH_SHORT).show();
+    mapView.getMapAsync(new OnMapReadyCallback() {
+      @Override
+      public void onMapReady(@NonNull final MapboxMap mapboxMap) {
+        IsochroneActivity.this.mapboxMap = mapboxMap;
+
+        mapboxMap.setStyle(Style.LIGHT, new Style.OnStyleLoaded() {
+          @Override
+          public void onStyleLoaded(@NonNull Style style) {
+            mapboxMap.addOnMapClickListener(IsochroneActivity.this);
+            initIsochroneCenterSymbolLayer();
```

Comment:
provide style as parameter to this method

Reply:
fixed